### PR TITLE
[libqcow] Initial port

### PR DESCRIPTION
**Pull request by VelocityRa**
Fixes build in #6852.

### CODE REVIEW

File: ports/libqcow/CONTROL [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Source: libqcow
+Version: 2018-12-27
```
Comment (NancyLi1013) [inline ports/libqcow/CONTROL:2]: The version should be `20191221`.

File: scripts/ci.baseline.txt [hunk 2]
```diff
@@ -876,6 +876,9 @@ libpng-apng:x64-windows-static = skip
 libpng-apng:x86-windows        = skip
 libpq:arm-uwp=fail
 libpq:x64-uwp=fail
+libqcow:arm-uwp=fail
+libqcow:x64-uwp=fail
```
Comment (NancyLi1013) [inline scripts/ci.baseline.txt:880]: Does this port not support `uwp`?
If so, please also add `Supports: !uwp` to CONTROL file and add `vcpkg_fail_port_install(ON_TARGET "uwp")` to the top of portfile.cmake.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by VelocityRa** [comment]
Done.

**Comment by VelocityRa** [comment]
CI failed because of something unrelated again...

**Comment by VelocityRa** [comment]
All tests pass, not sure why macOS is showing as failing on GitHub.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
osx pipeline seems to be moved to `microsoft.vcpkg (x64-osx)`.
So we can ignore the result on `vcpkg-osx-PR-test`.

**Comment by strega-nil** [comment]
Awesome, thanks @VelocityRa :)

**Pull request merged**